x-pack/plugins/alerts/server/alerts_client/alerts_client.ts

```diff
@@ -388,12 +395,47 @@ export class AlertsClient {
 
     logSuccessfulAuthorization();
 
-    return {
+    const findResults: FindResult = {
       page,
       perPage,
       total,
       data: authorizedData,
     };
+
+    // Replace this in the future when saved objects support aggregations
```

Comment:
I think it's kosher to add a comment with the link to the relevant PR (I don't see an issue for this) - https://github.com/elastic/kibana/pull/64002

Reply:
Added link.